pkg/controller/dnszone/dnszone_controller_test.go

```diff
@@ -1094,3 +1103,9 @@ func testInvalidSignatureExceptionError() error {
 		fmt.Errorf("The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details"))
 	return invalidSignatureErr
 }
+
+func newScheme() *runtime.Scheme {
+	scheme := runtime.NewScheme()
+	hivev1.AddToScheme(scheme)
+	return scheme
+}
```

Comment:
This appears vestigial. I took it out and the tests still pass. Did I miss something?

Reply:
Yup thanks, this was from my past impl, will remove.